core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/ConsoleOutputPaneModel.java

```diff
@@ -164,8 +164,8 @@ String createLine(MessageWithHeader m) {
         if (showHeaders) {
             sb.append(m.getHeader().toString(showUser,
                     showOrigin,
-                    showLevel,
                     showDate,
+                    showLevel,
                     showCode,
                     showThread1,
```

Comment:
What is the difference between thread1 and thread2 ?

Reply:
one is the hash and other the name of the thread.
I renamed the vars in most locations. will also do it here